lib/rules/require-unicode-regexp.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Rule to
+ * @author Toru Nagashima
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const {
+    CALL,
+    CONSTRUCT,
+    ReferenceTracker,
+    getStringIfConstant
+} = require("eslint-utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "enforce the use of `u` flag on RegExp",
+            category: "Best Practices",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/require-unicode-regexp"
+        },
+        messages: {
+            requireUFlag: "Require 'u' flag."
```

Comment:
I think this could be phrased better:

* `"'u' flag is required."`
* `Use the 'u' flag."`